setup.py

```diff
@@ -513,6 +513,31 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 # End dependencies group
 
+# Mypy 0.900 and above ships only with stubs from stdlib so if we need other stubs, we need to install them
+# manually as `types-*`. See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
+# for details. Wy want to install them explicitly because we want to eventually move to
+# mypyd which does not support installing the types dynamically with --install-types
+mypy_dependencies = [
+    'mypy',
```

Comment:
Should we pin/restrict the mypy version? I think we had problems in the past where mypy would update and change behaviour in a few places.

Reply:
Yes. I think I lost it when rebasing.